### ISSUE TITLE
Typos in `docs/user/basic_controls.rst`

### ISSUE THREAD

**Issue by dschiller** [hobyst/flmidi-kompletekontrol#26]
Typo in `docs/user/basic_controls.rst` at:

```rst
PLAY: Toogle play and pause (will light up when playback is active)
```

It should be `Toggle` instead of `Toogle`.

And

```rst
Restart: Toggle bewteen Pattern playback and Song playback
```

It should be `between` instead of `bewteen`.

And

```rst
REC: Toogle recording on/off (will light up when enabled)
```

It should be `Toggle` instead of `Toogle`.

**Comment by neshvig10** [comment]
I would like to work on this issue

**Comment by hobyst** [comment]
Fixed in 70f1d72.
